sktime/utils/__init__.py

```diff
@@ -9,6 +9,18 @@
 from pathlib import Path
 
 
+# We use estimator tags in addition to class hierarchies to further distinguish
+# estimators into different categories. This is useful for defining and running
+# common tests for estimators with the same tags.
+VALID_ESTIMATOR_TAGS = (
```

Comment:
Since we're not checking VALID_ESTIMATOR_TAGS in `_has_tag(...)`, could we move them back to the `test/_config.py` file or are there still circular imports?